denonavr/denonavr.py

```diff
@@ -278,6 +278,15 @@ def power(self) -> Optional[str]:
         """
         return self._device.power
 
+    @property
+    def settings_menu(self) -> Optional[str]:
+        """
+        Return the settings menu state of the device.
+
+        Possible values are: "ON" and "OFF"
+        """
+        return self._device.settings_menu
```

Comment:
Please add to the comment that this function only returns a value when telnet is enabled.